saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1067 @@
+        order_instance.created_at = order_input["created_at"]
+        order_instance.status = order_input["status"]
+        order_instance.user = instances.user
+        order_instance.billing_address = instances.billing_address
+        order_instance.shipping_address = instances.shipping_address
+        order_instance.language_code = order_input["language_code"]
+        order_instance.user_email = instances.user.email
+        order_instance.collection_point = delivery_method.warehouse
+        order_instance.collection_point_name = delivery_input.get(
+            "warehouse_name"
+        ) or getattr(delivery_method.warehouse, "name", None)
+        order_instance.shipping_method = delivery_method.shipping_method
+        order_instance.shipping_method_name = delivery_input.get(
+            "shipping_method_name"
+        ) or getattr(delivery_method.shipping_method, "name", None)
+        order_instance.shipping_tax_class = delivery_method.shipping_tax_class
+        order_instance.shipping_tax_class_name = delivery_input.get(
+            "shipping_tax_class_name"
+        ) or getattr(delivery_method.shipping_tax_class, "name", None)
+        order_instance.shipping_tax_rate = order_amounts.shipping_tax_rate
+        order_instance.shipping_price_gross_amount = order_amounts.shipping_price_gross
+        order_instance.shipping_price_net_amount = order_amounts.shipping_price_net
+        order_instance.total_gross_amount = order_amounts.total_gross
+        order_instance.undiscounted_total_gross_amount = (
+            order_amounts.undiscounted_total_gross
+        )
+        order_instance.total_net_amount = order_amounts.total_net
+        order_instance.undiscounted_total_net_amount = (
+            order_amounts.undiscounted_total_net
+        )
+        order_instance.customer_note = order_input.get("customer_note", "")
+        order_instance.redirect_url = order_input.get("redirect_url")
+        order_instance.origin = OrderOrigin.BULK_CREATE
+        order_instance.weight = order_input.get("weight", zero_weight())
+        order_instance.tracking_client_id = order_input.get("tracking_client_id")
+        order_instance.currency = order_input["currency"]
+        order_instance.should_refresh_prices = False
+        update_order_display_gross_prices(order_instance)
+
+        if metadata := delivery_method.shipping_tax_class_metadata:
+            for data in metadata:
+                order_instance.shipping_tax_class_metadata.update(
+                    {data["key"]: data["value"]}
+                )
+        if private_metadata := delivery_method.shipping_tax_class_private_metadata:
+            for data in private_metadata:
+                order_instance.shipping_tax_class_private_metadata.update(
+                    {data["key"]: data["value"]}
+                )
+
+        # TODO charged
+        # TODO authourized
+        # TODO voucher
+        # TODO gift cards
```

Comment:
What action should be taken regarding this TODO?

Reply:
This will be handled in separate PRs - description updated.